universal-login-react/src/ui/WalletSelector/WalletSelector.tsx

```diff
@@ -15,10 +16,11 @@ import ethLogo from '../assets/icons/ethereum-logo.svg';
 import './../styles/walletSelector.css';
 import './../styles/walletSelectorDefaults.css';
 import './../styles/hint.css';
+import {MissingParameter} from '../../core/utils/errors';
 
 interface WalletSelector {
-  onCreateClick(ensName: string): Promise<void> | void;
-  onConnectClick(ensName: string): Promise<void> | void;
+  onCreateClick?(ensName: string): Promise<void> | void;
```

Comment:
It looks strange. Make it `onCreateClick?: (ensName: string) => Promise<void>`

Reply:
I did it for the developer's convience

Reply:
👍